tools/torch_quant/tests/test_quantizer.py

```diff
@@ -113,6 +115,82 @@ def test_calib_quantize_qat_quantize_state_equal(self):
         out3 = quant_model(dummy_input)
         self.assertTrue(torch.equal(out2, out3))
 
+    def _test_calib_and_quantize_with_op_types_filter(
+        self, backend: Backend, module_filter: ModuleFilter
+    ) -> None:
+        model = SimpleModule()
+        quantizer = Quantizer(backend=backend, module_filter=module_filter)
+        dummy_input = torch.randn((1, 2, 5, 5))
+        original_output = model(dummy_input)
+
+        calib_model = quantizer.calib(model)
+        calib_output = calib_model(dummy_input)
+        self.assertTrue(torch.equal(original_output, calib_output))
+
+        quant_model = quantizer.quantize(model)
+        self.assertTrue(isinstance(quant_model.conv, torch.nn.Conv2d))
+        self.assertTrue(isinstance(quant_model.sub.conv, torch.nn.Conv2d))
+        self.assertEqual(quant_model.conv, model.conv)
+        self.assertEqual(quant_model.sub.conv, model.sub.conv)
+        self.assertNotEqual(quant_model.linear, model.linear)
+
+        quant_output = quant_model(dummy_input)
+        self.assertFalse(torch.equal(original_output, quant_output))
+        torch.testing.assert_close(
+            quant_output, original_output, rtol=0.1, atol=0.5)
+
+    @parameterized.expand([(Backend.REFERENCE, ), (Backend.DISC, )])
```

Comment:
no need to test different backends for type filter? And, maybe re-org include/exclude as parameterized

Reply:
Done. Test different backends maybe more reliable , but really no need to test all different combinations.